Makefile

```diff
@@ -39,10 +39,15 @@ install-all: install install-tools-testing # Install everything for and of this
 .PHONY: install-all
 
 install-tools-testing: # Install tools that are used for testing.
+	go install -v github.com/vektra/mockery/v2@v2.42.3
 	go install -v gotest.tools/gotestsum@v1.11.0
 	eval-dev-quality install-tools
 .PHONY: install-tools-testing
 
-test: # [<Go package] - # Test everything, or only the specified package.
+generate: # Run code generation.
+	mockery
+.PHONY: generate
+
+test: generate # [<Go package] - # Test everything, or only the specified package.
```

Comment:
Please add a CI check that fails if we the generation is not up to date. Y ou can copy that from our internal lint-build CI job.